client/me/purchases/manage-purchase/index.jsx

```diff
@@ -932,7 +931,7 @@ export default connect( ( state, props ) => {
 		isProductOwner,
 		site,
 		renewableSitePurchases,
-		plan: isPurchasePlan && applyTestFiltersToPlansList( purchase.productSlug, abtest ),
+		plan: isPurchasePlan && applyTestFiltersToPlansList( purchase.productSlug, undefined ),
```

Comment:
The `abtest` param should be eventually removed from `applyTestFiltersToPlansList` and related API of the `calypso-products` package, is that right?

Reply:
That's right. I consider this is work for another PR though because it needs to touch other places that include that argument that this PR is not touching.